app/util/mapIconUtils.js

```diff
@@ -70,6 +71,15 @@ export function drawRoundIcon(tile, geom, type, large) {
       );
 
       tile.ctx.fill();
+
+      if (platformNumber && hubRadius > 7) {
```

Comment:
Instead of using "7", I would either:
- pass empty platformNumber to this function if z-level is not what is configured in config; or
- pass z-level to this function and compare against configured value in config.

in any case number 7 should IMHO be in config